src/Microsoft.VisualStudio.ProjectSystem.VisualBasic.VS/ProjectSystem/VS/Automation/VSImports.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.ComponentModel.Composition;
+using System.Linq;
+using EnvDTE;
+using Microsoft.VisualStudio.ProjectSystem.VS.ConnectionPoint;
+using VSLangProj;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Automation
+{
+    [Export(typeof(Imports))]
+    [Export(typeof(ImportsEvents))]
+    [AppliesTo(ProjectCapability.VisualBasic)]
```

Comment:
You mean if only one import of that type is expected, then capabilities are not honored?

Would exporting this type with Order and Import Many with OrderPrecedenceCollection work? or is there a better way to do this?